managed/utils/interceptors/interceptors.go

```diff
@@ -118,10 +120,12 @@ func Stream(srv interface{}, ss grpc.ServerStream, info *grpc.StreamServerInfo,
 	}
 	ctx = logger.SetEntry(ctx, l)
 
+	ctx = SetCallerOrigin(ctx, info.FullMethod)
```

Comment:
Do I understand properly that now managed's custom interceptor depends on prometheus's ones? So in case we commit prometheus interceptor during gRPC server configuration this function call is useless, right?

Reply:
go-grpc-prometheus is a brigde between gRPC and Prometheus metrics, we had go-grpc-prometheus working as gRPC interceptor, but now we need to provide extra data for custom labels